massa-final-state/src/final_state.rs

```diff
@@ -16,6 +16,7 @@ use massa_db_exports::{
     EXECUTED_OPS_PREFIX, LEDGER_PREFIX, MIP_STORE_PREFIX, STATE_CF,
 };
 use massa_db_exports::{EXECUTION_TRAIL_HASH_PREFIX, MIP_STORE_STATS_PREFIX, VERSIONING_CF};
```

Comment:
general remark: we also need to handle network restarts.

Subquestion: what happens to deferred calls that were to be executed during the downtime ?